frontends/mit-open/src/pages/DepartmentListingPage/DepartmentListingPage.tsx

```diff
@@ -0,0 +1,232 @@
+import React from "react"
+import {
+  Container,
+  Typography,
+  styled,
+  PlainList,
+  List,
+  ListItem,
+  ListItemLink,
+  ListItemText,
+} from "ol-components"
+import type { TypographyProps } from "ol-components"
+import { MetaTags, pluralize } from "ol-utilities"
+import type {
+  LearningResourceSchool,
+  LearningResourceSearchResponse,
+} from "api"
+import {
+  useLearningResourcesSearch,
+  useSchoolsList,
+} from "api/hooks/learningResources"
+import {
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiBriefcaseLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiArrowRightSLine,
+} from "@remixicon/react"
+
+const SCHOOL_ICONS: Record<string, React.ReactNode> = {
```

Comment:
Very open to better ideas for tying icons to the schools returned by https://mitopen-rc.odl.mit.edu/api/v1/schools/

Reply:
If their IDs (or name even) are less likely to change than their URLs, it would be better to use those. Alternatively, as these are icons and not logos, we could match on e.g. "Architecture", "Engineering", "Humanities" in the name.

Reply:
I don't feel great about using the numeric ids, even though they are specified in the fixture file that populates the Schools. The URL seemed most stable to me.

We're going to have a similar "Topics Listing" page. In that case, we'll definitely want a different solution, probably picking the icon based on some API parameter.